src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:34789/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:34789/project/issues?id=sample&branch=feature1&issues=AWDhghaZQEf3q6O3TqDE&open=AWDhghaZQEf3q6O3TqDE)

[ISSUE_KEY]: AWDhghaZQEf3q6O3TqDE